backend/src/main/java/com/google/sps/servlets/RemoveParticipantServlet.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.sps.RemoveParticipantHelper;
+import com.google.sps.UsernameService;
+import com.google.sps.datastore.ParticipantDatastore;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that removes a participant from the queue */
+@WebServlet("/api/v1/remove-participant")
+public class RemoveParticipantServlet extends HttpServlet {
+
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private final ParticipantDatastore participantDatastore = new ParticipantDatastore(datastore);
+  private final UsernameService usernameService = new UsernameService();
+  private final RemoveParticipantHelper helper =
+      new RemoveParticipantHelper(participantDatastore, usernameService);
```

Comment:
I put this in this class's doPost method with request/response parameters too, and had the helper doPost method take in request, response, datastores, and usernameService. Whatever we decide, we should keep it consistent across the servlets. Thoughts?

Reply:
The rationale for taking this out of the doPost method was so that the servlet could instantiate this once and reuse it. I think this would help in future for scalability reasons, what do you think?